dm/dm/worker/subtask.go

```diff
@@ -83,6 +75,14 @@ func createRealUnits(cfg *config.SubTaskConfig, etcdClient *clientv3.Client, wor
 	return us
 }
 
+func newLoadUnit(cfg *config.SubTaskConfig, etcdClient *clientv3.Client, workerName string) unit.Unit {
+	// tidb-lightning doesn't support column mapping currently
+	if cfg.ImportMode == config.LoadModeLoader || cfg.OnDuplicate == config.OnDuplicateError || len(cfg.ColumnMappingRules) > 0 {
```

Comment:
maybe this check should added to `func (m *LoaderConfig) adjust() ` and return an error if user set import mode is `sql` with `ColumnMappingRules`

Reply:
In most case, we expect our user just use the "default" value and don't need to pay much attention for whether we user loader or lightning to execute SQL. Thus, lightning can't support some features is our internal restriction and we should auto adjust it instead of report an error to our user especially when user just set the default config.